### DISCUSSION TITLE
Faulting application name: SearchIndexer.exe; Faulting module name: ExplorerPatcher.amd64.dll

### DISCUSSION THREAD

**Discussion by HuckFinn2021** [valinet/ExplorerPatcher#297] [Q&A]
Any ideas how to fix  this:

Faulting application name: SearchIndexer.exe, version: 7.0.22000.282, time stamp: 0x5759ba2c
Faulting module name: ExplorerPatcher.amd64.dll, version: 22000.258.31.4, time stamp: 0x6172fb61
Exception code: 0xc0000005
Fault offset: 0x0000000000006cc0
Faulting process id: 0x1e38
Faulting application start time: 0x01d7d45f507a33a1
Faulting application path: C:\WINDOWS\system32\SearchIndexer.exe
Faulting module path: C:\Program Files\ExplorerPatcher\ExplorerPatcher.amd64.dll
Report Id: a51fac93-f248-492d-af23-ada2e92625fb
Faulting package full name: 
Faulting package-relative application ID:
___________________________________________________________

All the permissions  are checked/enabled except for special permissions for all 3 files in \ExplorerPatcher\. category.
I've put Program Files in the Search Index excluded category.

**Comment by valinet** [comment]
Have you registered it as shell extension?

**Comment by HuckFinn2021** [comment]
I believe I ran regsvr32 ExplorerPatcher.amd64.dll when I initially put 2 DLLs (ExplorerPatcher.amd64.dll and ExplorerPatcher.IA-32.dll) in "C:\Program Files\ExplorerPatcher". To be sure, I ran it again with the same result.  This time I noticed a warning that may be meaningful.

 

Repeating Fault

Faulting application name: SearchIndexer.exe, version: 7.0.22000.282, time stamp: 0x5759ba2c

Faulting module name: ExplorerPatcher.amd64.dll, version: 22000.258.31.4, time stamp: 0x6172fb61

Exception code: 0xc0000005

Fault offset: 0x0000000000006cc0

Faulting process id: 0x1e38

Faulting application start time: 0x01d7d45f507a33a1

Faulting application path: C:\WINDOWS\system32\SearchIndexer.exe

Faulting module path: C:\Program Files\ExplorerPatcher\ExplorerPatcher.amd64.dll

Report Id: a51fac93-f248-492d-af23-ada2e92625fb

Faulting package full name: 

Faulting package-relative application ID:

 

Newly noticed Warning 

The application-specific permission settings do not grant Local Activation permission for the COM Server application with CLSID 

{6B3B8D23-FA8D-40B9-8DBD-B950333E2C52}

and APPID 

{4839DDB7-58C2-48F5-8283-E1D1807D0D7D}

to the user NT AUTHORITY\LOCAL SERVICE SID (S-1-5-19) from address LocalHost (Using LRPC) running in the application container Unavailable SID (Unavailable). This security permission can be modified using the Component Services administrative tool.

 

 

From regedit: {6B3B8D23-FA8D-40B9-8DBD-B950333E2C52 = ShellServiceHost

 

I then did the uninstall (regsvr32 /u ExplorerPatcher.amd64.dll), reboot, and search indexing appears to be working.

 

I’ll do more experiments if you tell me what to do, but for the time being all is well.

 

 

 

__________________________________________________________________

Paul Edmonds
Redwood City, CA 94061
***@***.*** ***@***.***> 
mobile: 650-283-5125

**Reply by valinet** [reply]
Yeah, registering as shell extension is not required, as I said a fair couple of times by now, to use 99% of the features offered by EP. That' why it is not mentioned anywhere, it's only good for specific uses one should already know about and even then, it is still experimental and a work in progress pretty much.

Thanks
